README.md

```diff
@@ -38,9 +44,9 @@ Installation
 ------------
 
 You can install random_useragent by running the following command:
-
-    pip install random_user_agent
-
+```bash
```

Comment:
Please revert this change. The developers will be installing the package from the PyPI and this command will not install the intended version of the package.

Reply:
Sowy I will, I just wanted ppl to be able to clone my fork.